src/components/UserInfo/UserInfor.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
```

Comment:
you don't have to add empty lines between import
```suggestion
import React from 'react';
import './UserInfo.scss';
import { User } from '../../types/User';
```